### PR TITLE
Add patches and configuration for Etherpad InstantApp

### PR DESCRIPTION
**Pull request by edouardb**

### CODE REVIEW

File: patches/etc/update-motd.d/70-etherpad [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+PUBLIC_IP=$(oc-metadata --cached PUBLIC_IP_ADDRESS)
+
+cat <<EOF
+Hi Cloud Rider,
+Welcome to the Etherpad image!
+Redmine is installed in /opt/etherpad.
```
Comment (moul) [inline patches/etc/update-motd.d/70-etherpad]: Redmine

File: patches/root/README [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Hi Cloud Rider,
+Welcome to the Etherpad image!
+
+Etherpad is installed in /opt/etherpad.
+Log into Redmine at http://your-ip:9001.
```
Comment (moul) [inline patches/root/README]: Redmine

File: patches/usr/local/sbin/oc-init-etherpad [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+MYSQL_DB_NAME="etherpad"
+
+# MOTD & README
+sed -i "s/{mysql_root_password}/$MYSQL_ROOT_PASSWORD/" /etc/update-motd.d/70-etherpad
+sed -i "s/{mysql_root_password}/$MYSQL_ROOT_PASSWORD/" /root/README
+
+# Configure etherpad database
+
+sed -i "s/{MYSQL_DB_PASSWORD}/$MYSQL_ROOT_PASSWORD/" /opt/etherpad/settings.json
+sed -i "s/{MYSQL_DB_NAME}/$MYSQL_DB_NAME/" /opt/etherpad/settings.json
```
Comment (moul) [inline patches/usr/local/sbin/oc-init-etherpad]: This can be hardcoded in the settings.json file

**Comment by moul** [comment]
Only small notes, so LGTM

Can you fixup your **typo** commits ?

File: patches/usr/local/sbin/oc-init-etherpad (hunk 3 above)
Comment (edouardb) [inline patches/usr/local/sbin/oc-init-etherpad]: I firstly tried but it didn't work.

**Comment by edouardb** [comment]
Done!

**Comment by moul** [comment]
Perfect !

**Pull request merged**